src/ui/RealmBrowser/Content/Table/types/StringCell/StringCell.tsx

```diff
@@ -51,6 +51,7 @@ export const StringCell = ({
           'RealmBrowser__Table__StringCell--null': value === null,
         },
       )}
+      spellcheck={property.type === 'string' ? 'true' : 'false'}
```

Comment:
Not sure if we should disable it altogether (a red squiggly line can be missread as a cells error-state). For now left in for 'string' types.